fixes.inc

```diff
@@ -1877,1013 +1877,1013 @@ in the output code.
 #define _FIXES_IS_UNSET(%0) ((2*%0-1+1)==-1)
 
 #if !defined FIXES_EnableAll
-	#define FIXES_EnableAll                      (0)
+	#define FIXES_EnableAll						(0)
```

Comment:
These should be spaces.  Tabs are used for indentation, spaces are used for in-line alignment.